docs/developer-tools/ts-sdk-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Marlowe TypeScript SDK
+sidebar_position: 2
+---
+
+## Introduction
+The **[Marlowe TypeScript SDK (TS-SDK)](https://github.com/input-output-hk/marlowe-ts-sdk/)** consists of JavaScript and TypeScript libraries. It's engineered to support DApp developers with the tools they need to build and integrate with the Marlowe smart contract ecosystem on the Cardano blockchain. 
+
+## Features at a glance
+
+1. **Smart contract toolkit**: Craft, deploy, and manage Marlowe smart contracts on the Cardano blockchain with the tools and libraries in the TS-SDK.
+2. **Integration with Marlowe Playground**: The TS-SDK works well with the Marlowe Playground, an online interface dedicated to designing, simulating, and scrutinizing Marlowe contracts.
+3. **Wallet connectivity**: With built-in modules, the SDK promotes smooth interactions with various wallet extensions. This ensures easy access to wallet data and the efficient integration of Marlowe contracts with a variety of wallet interfaces.
+4. **Proof-of-Concept (PoC) templates**: Embark on your DApp journey using the TS-SDK's distilled example templates. These PoC templates serve as invaluable launching pads for your customized applications.
+
+## Getting started
+To use the capabilities of the Marlowe TS-SDK, have an active link to a Marlowe Runtime instance and a browser-installed wallet extension that's compatible.
```

Comment:
What does an "active link" to a Marlowe Runtime instance mean?

Reply:
I don't know what it is also... We should have a "How To Get this Active Link"

Reply:
I'm just referring to a URL that points to a running instance of the Runtime. I'll make this more clear.